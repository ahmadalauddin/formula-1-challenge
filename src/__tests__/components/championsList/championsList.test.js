import React from "react";
import {shallow} from "enzyme";
import ChampionsList from "../../../components/championsList/championsList";

let wrapper;

beforeEach(()=>{
  wrapper = shallow(<ChampionsList/>);
  
})

describe("Testing champions list component", () => {
  it("Checking the snapshot of the component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
