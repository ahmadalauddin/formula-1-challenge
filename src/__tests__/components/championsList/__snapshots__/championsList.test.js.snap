// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing champions list component Checking the snapshot of the component 1`] = `
<div
  className="container d-flex justify-content-center align-items-center"
  style={
    Object {
      "height": "60vh",
    }
  }
>
  <Loader
    color={
      Object {
        "100": "#c8e6c9",
        "200": "#a5d6a7",
        "300": "#81c784",
        "400": "#66bb6a",
        "50": "#e8f5e9",
        "500": "#4caf50",
        "600": "#43a047",
        "700": "#388e3c",
        "800": "#2e7d32",
        "900": "#1b5e20",
        "A100": "#b9f6ca",
        "A200": "#69f0ae",
        "A400": "#00e676",
        "A700": "#00c853",
      }
    }
    css=""
    loading={true}
    size={100}
    speedMultiplier={1}
  />
</div>
`;
